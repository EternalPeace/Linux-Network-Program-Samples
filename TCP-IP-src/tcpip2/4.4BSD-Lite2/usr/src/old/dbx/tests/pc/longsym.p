program symtest (input, output);

procedure Level1Procedure ;
    procedure Level2Procedure ;
	procedure Level3Procedure ;
	    procedure Level4Procedure ;
		procedure Level5Procedure ;
		begin
		end;
	    begin
	    end;
	begin
	end;
    begin
    end;
begin
end;

begin
end.
