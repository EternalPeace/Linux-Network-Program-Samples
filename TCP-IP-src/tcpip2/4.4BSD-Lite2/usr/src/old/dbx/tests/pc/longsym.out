reading symbolic information ...
.Level1Procedure
Level1Procedure.Level2Procedure
Level1Procedure.Level2Procedure.Level3Procedure
Level1Procedure.Level2Procedure.Level3Procedure.Level4Procedure
Level1Procedure.Level2Procedure.Level3Procedure.Level4Procedure.Level5Procedure

